soco/core.py

```diff
@@ -1369,6 +1369,9 @@ def add_to_queue(self, queueable_item, position=0, as_next=False):
                 added. Default is 0 (add URI at the end of the queue).
             as_next (bool): Whether this URI should be played as the next
                 track in shuffle mode. This only works if `play_mode=SHUFFLE`.
+                
+        Returns:
+            An integer representing the index of the new item in the queue.
```

Comment:
To comply with the Napoleon docstring style, this should be
`int: The index of the new item in the queue.`